cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/NameFilterType.java

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
I don't know what's the best location to put this enum file

Reply:
should be wher ApplicationFilter is?